content/03.results.md

```diff
@@ -224,46 +235,47 @@ The mutual exclusivity of signatures 3 and MMR corroborates a previous report su
 #### Histologic and oncogenic pathway clustering
 
-UMAP visualization of gene expression variation across brain tumors (**Figure {@fig:Fig5}A**) shows the expected clustering of brain tumors by histology.
-We observed medulloblastomas cluster by molecular subtype with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (**Figure {@fig:S6}A**), as expected.
-Of note, two samples annotated as the SHH subtype do not cluster with the MB samples and one clusters with Group 3 and 4 samples, suggesting potential subtype misclassification or different underlying biology of these tumors.
-
-Additionally, except for three outliers, _C11orf95::RELA_ (_ZFTA::RELA_) fusion positive ependymomas fall within distinct clusters (**Figure {@fig:S6}B**).
+UMAP visualization of gene expression variation across brain tumors (**Figure {@fig:Fig5}A**) showed the expected clustering of brain tumors by histology.
+We additionally explored UMAP projections of gene expression within molecular subtypes for certain cancer groups. 
+We observed that, except for three outliers, _C11orf95::RELA_ (_ZFTA::RELA_) fusion-positive ependymomas fell within distinct clusters (**Figure {@fig:S6}A**).
+Medulloblastoma (MB) samples cluster by molecular subtype, with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (**Figure {@fig:S6}B**), as expected.
+Of note, two MB samples annotated as the SHH subtype did not cluster with the other MB samples, and one clustered with Group 3 and 4 samples, suggesting potential subtype misclassification or different underlying biology of these two tumors.
 _BRAF_-driven low-grade gliomas (**Figure {@fig:S6}C**) were present in three separate clusters, suggesting that there might be additional shared biology within each cluster.
-Histone H3 G35-mutant HGGs generally clustered together and away from K28-mutant tumors (**Figure {@fig:S6}D**).
-Interestingly, although H3 K28-mutant tumors have different biological drivers than H3 wildtype tumors, they did not form distinct clusters. 
-This suggests they may be driven by common transcriptional programs, have other much stronger biological drivers such as their known distinct epigenetic drivers, or our sample size is too small to detect transcriptional differences.
+Histone H3 G35-mutant high-grade gliomas generally clustered together and away from K28-mutant tumors (**Figure {@fig:S6}D**).
+Interestingly, although H3 K28-mutant tumors have different biological drivers than do H3 wildtype tumors **TODO: Is there a reference for different biological drivers?**, they did not form distinct clusters. 
+This pattern suggests these subtypes may be driven by common transcriptional programs, have other much stronger biological drivers such as their known distinct epigenetic drivers **TODO: this phrase confuses me because I don't see how _distinct_ epigenetic drivers would cause clustering together?** , or our sample size is too small to detect transcriptional differences.
```

Comment:
Ah, this is because of the origin of the H3 K28 mutation - it turns H3K27Me3 (repressive) to H3K27Ac (enhancer), thus more open chromatin, more accessibility for transcription to occur. I would have expected H3 wildtype and K28s to cluster apart based on this mutation, which causes an epigenetic modification, since the chromatin landscape of the tumors is _so_ drastically different. Are you saying that you don't understand why they would cluster together given this above diff? That is why it is surprising to me and this is just a list of possibilities, one of which is small sample size to detect a real effect.

Also, I think I should have said this:
```suggestion
This pattern suggests these subtypes may be driven by common transcriptional programs, have other much stronger biological drivers than their known distinct epigenetic drivers, or our sample size is too small to detect transcriptional differences.
```